### ISSUE TITLE
Battery level is zero `C0% P0%`

### ISSUE THREAD

**Issue by alloc33** [Katona/zmk-ble#2]
Is there any solution how to fix this?
Attached conf file. (I use Miryoku)

<img width="283" alt="image" src="https://user-images.githubusercontent.com/45129790/213896509-12e495d1-e113-4f82-84b2-d9b13effd7af.png">

```
CONFIG_ZMK_KEYBOARD_NAME="Chocofi ZMK"
CONFIG_USB_DEVICE_VID=0x13BA
CONFIG_USB_DEVICE_PID=0x3E7
CONFIG_ZMK_BLE=y
CONFIG_BT=y
CONFIG_ZMK_IDLE_TIMEOUT=30000
CONFIG_ZMK_SLEEP=y
CONFIG_ZMK_IDLE_SLEEP_TIMEOUT=900000
CONFIG_ZMK_MOUSE=y
```
My config is based on ZMK repo main branch + another branches which were merged into the main:
```ftc/zmk/mouse-ftc petejohanson/zmk/ble/gatt-dis-config-improvements manna-harbour/zmk/suspend-test```

**Comment by Katona** [comment]
Hi, you need https://github.com/zmkfirmware/zmk/pull/1243 to be merged for this to work.

**Comment by alloc33** [comment]
@Katona It's doesn't work anymore. I'm using miryoku layout and github actions
<img width="637" alt="image" src="https://github.com/Katona/zmk-ble/assets/45129790/623eced4-e777-469e-aa46-9e955643b6ca">

**Comment by Katona** [comment]
Yeah, my branch is completely outdated, I didn't get the chance to update it.

**Comment by alloc33** [comment]
@Katona could you please give a hint what should be updated?

**Comment by equiman** [comment]
It was already finish and merged to main.

https://github.com/zmkfirmware/zmk/pull/2045#event-11381541407
